src/utils/config.ts

```diff
@@ -0,0 +1,36 @@
+import { createPublicClient } from 'viem';
```

Comment:
Interesting, so we're going to do a dev network for this? 

we don't currently have anything deployed to a dev network atm.